### PR TITLE
Add dock sensors

### PR DESCRIPTION
**Pull request by Lash-L**
Wanted to just add sensors and since there were so many changes, I just wanted a clean start so I can build this on top of it and leave the last branch unaffected if I add services down the line.

### CODE REVIEW

**Comment by humbertogontijo** [comment]
Is this open for review?

**Comment by humbertogontijo** [comment]
I do not have it either. I will review it as it is and we can make the necessary changes

File: custom_components/roborock/sensor.py [hunk 1]
```diff
@@ -185,56 +190,86 @@ class RoborockSensorDescription(SensorEntityDescription):
...
         parent_key=RoborockDevicePropField.CONSUMABLE,
-        name="Sensor dirty left",
+        translation_key="sensor_dirty_left",
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    f"current_{ATTR_DOCK_STATUS}": RoborockSensorDescription(
+        key="dock_error_status",
+        icon="mdi:garage-open",
+        parent_key=RoborockDevicePropField.STATUS,
+        translation_key="dock_status",
```
Comment (humbertogontijo) [inline custom_components/roborock/sensor.py:240]: And add for the new ones

File: custom_components/roborock/sensor.py [hunk 2]
```diff
@@ -104,16 +109,16 @@ class RoborockSensorDescription(SensorEntityDescription):
         icon="mdi:clock-time-twelve",
         device_class=SensorDeviceClass.TIMESTAMP,
         parent_key=RoborockDevicePropField.LAST_CLEAN_RECORD,
-        name="Last clean end",
+        translation_key="last_clean_end",
```
Comment (humbertogontijo) [inline custom_components/roborock/sensor.py:112]: Can you keep name property for all decriptions?

**Review by humbertogontijo — CHANGES_REQUESTED** [review]

**Comment by Lash-L** [comment]
fixed

**Review by humbertogontijo — APPROVED** [review]

**Comment by Menelao147** [comment]
Hi, I'm trying to translate the descriptions of the new sensor. Can you explain what they mean?

**Comment by Lash-L** [comment]
@Menelao147 

dock_status is the status for docks. Ends up saying "okay" or "water low" or other things like that

wash_towel_mode - is the mode it will use to clean the mopping pad on the robot

dust collection mode - is how it will get the dust out of the robot. Basically how strong it will try to suction it out/ how long it will suction for.

dock_mop_wash_mode_interval - is the frequency at which the robot will return to the dock to get the mopping pad cleaned

**Pull request merged**